src/managing-workflow/tuning-component-settings.md

```diff
@@ -19,6 +19,22 @@ env:
     value: "admin_only"
 ```
 
+## Setting Resource limits
+You can set resource limits to workflow components by modifying the template file `tpl/generate_params.toml`.
+This file has a section for each workflow component to set a limit to any workflow component just add limits_cpu, limits_memory
```

Comment:
We should break this sentence up, and replace "your" with "the".

"This file has a section for each workflow component to set a limit to any workflow component. Just add `limits_cpu` and `limits_memory` in the section and set them to the appropriate values."